manifestival.go

```diff
@@ -131,9 +131,9 @@ func (m Manifest) apply(spec *unstructured.Unstructured, opts ...ApplyOption) er
 	}
 	if current == nil {
 		m.logResource("Creating", spec)
+		annotate(spec, "manifestival", resourceCreated)
 		current = spec.DeepCopy()
 		annotate(current, v1.LastAppliedConfigAnnotation, lastApplied(current))
-		annotate(current, "manifestival", resourceCreated)
```

Comment:
Try leaving both annotations on current and just add the annotate of the spec after...
```
		current = spec.DeepCopy()
		annotate(current, v1.LastAppliedConfigAnnotation, lastApplied(current))
		annotate(current, "manifestival", resourceCreated)
		annotate(spec, "manifestival", resourceCreated)
		return m.Client.Create(current, opts...)
```

Reply:
Great, thanks.